elasticsearch/reindex/progress.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Usage: [watch -t] ./progress.sh <current index doc count>
+
+DOC_COUNT=$1
```

Comment:
> we add 20 images and we get to 30/120

It'll still be 30/100. During re-index, we clone the index from a point in time. The changes are replayed during the syncing stage.